schemas/app-sre/automated-action-1.yml

```diff
@@ -16,29 +16,111 @@ properties:
     - /app-sre/automated-action-1.yml
   labels:
     "$ref": "/common-1.json#/definitions/labels"
-  operationId:
+  type:
     type: string
     description: |
-      The unique identifier for the automated action. This corresponds to 
-      the OpenAPI operation ID for the action.
+      The automated action type, which indicates the kind of operation
+      being performed.
   description:
     type: string
     description: |
-      A brief description of the automated action, explaining its purpose 
+      A brief description of the automated action, explaining its purpose
       and functionality.
+  instances:
+    type: array
+    items:
+      "$ref": "/common-1.json#/definitions/crossref"
+      "$schemaRef": "/app-sre/automated-actions-instance-1.yml"
+    description: |
+      A list of instances where the automated action can be executed.
   retries:
     type: number
     description: |
-      The number of times to retry the action in case of failure. This helps 
+      The number of times to retry the action in case of failure. This helps
```

Comment:
We should remove retries until we implement it (if we implement it, because it's completely outside OPA)